service/src/main/java/com/ge/predix/acs/privilege/management/EntityManagementControllerAdvice.java

```diff
@@ -26,4 +26,16 @@
                                     + " must be a boolean value");
                 }});
     }
+
+    @ExceptionHandler(ZoneDoesNotExistException.class)
```

Comment:
```EntityManagementControllerAdvice``` only applies to Exception handling to ```SubjectPrivilegeManagementController``` and ```ResourcePrivilegeManagementController```. What if the zone doesn't exist when someone with tries to get connector configuration information? This should probably be split out into it's own global controller.